scripts/README.md

```diff
@@ -0,0 +1,177 @@
+# Deployment scripts
+
+This directory contains two types of script: [operate test and discovery
+environments](../create-a-discovery-register-environment.md) and upload data
+to the Open Register Java (ORJ) application.
+
+
+## Introduction
+
+The raw data for every GDS-hosted register is in Github:
```

Comment:
this isn't the case currently. I don't think we should say this in the README until it reflects the current reality.

Reply:
Once this PR is merged it will be true (see linked PR)